external/tf.patch

```diff
@@ -87,7 +87,9 @@ index 00000000000..31ba58594cc
 +// `DVLOG` behaves like `VLOG` in debug mode (i.e. `#ifndef NDEBUG`).
 +// Otherwise, it compiles away and does nothing.
 +#ifndef NDEBUG
++#ifndef DVLOG
```

Comment:
Why do we need this patch?

Reply:
building in debug doesn't work otherwise